setup.py

```diff
@@ -203,6 +204,56 @@ def run(self):
             sys.exit(128)
 
 
+class Selftests(SimpleCommand):
+    """Run selftests"""
+
+    description = 'Run selftests'
+    user_options = [
+        ("job-api", None, "Run job API checks"),
+        ("static-checks", None, "Run static checks (isort, lint, etc)"),
```

Comment:
job-api and static-checks are also inside "selftests" category. Any reason for not using the same prefix here?

Reply:
The options naming are inherited from `selftests/check.py`, see https://github.com/avocado-framework/avocado/blob/master/selftests/check.py#L198

I fully agree about using the same prefix to all of them. In this case, given the context, isn't maybe better just remove the "selftest" from all of them?  They're always used  in a context that makes clear is a selftests, either with
`python3 setup.py selftest --static-checks`
or with:
`./selftests/check.py --static-checks`

Reply:
Sure, lets remove. I agree.